internal/provider/user_resource_test.go

```diff
@@ -74,6 +74,34 @@ func TestAccUserResource(t *testing.T) {
 	})
 }
 
+func TestAccUserResourceImport(t *testing.T) {
```

Comment:
Add import test as a different test suite because in the other one we run with `CheckDestroy` which somehow messes up with the test import. I didn't dig too much on it, because this works